app/assets/javascripts/admin/slides.js

```diff
@@ -149,6 +149,46 @@ AdminSlides.initSlideActionMenus = function() {
   });
 };
 
+
+/**
+ *  Ugly Temporary Add Owner Callback
+ *  This is a temporary fix to allow users to add owners to slides. The issue is connected to the jQuery autocomplete not working.
+ *  See GitHub issue: https://github.com/chapmanu/signage/issues/183
+ *  There is believed to be a naming conflict with jQuery autocomplete and the Materialize autocomplete function that was added in version 0.98.0
+ *  This fix may be able to be removed when Materialize fixes the naming issue on their end.
+ *  The same uglyTempAddOwnerCallback function is used in signs.js
+ */
+AdminSlides.uglyTempAddOwnerCallback = function(e) {
+  $('#add_owner').on('click', function(e) {
+
+    // stop all
+    e.preventDefault();
+
+    var user = $('#search_users').val();
+
+    $.ajax({
+      url: '/slides/autocomplete_user_email?term=' + user,
+      type: "GET",
+    }).done(function(returnedData) {
+      // if returnedData not null
+      if($.trim(returnedData)){
+        returnedData.forEach(function(element) {
+
+          //check for exact match email, exclude partials
+          if(element.email === user){
+            $('input#user_id').val(element.id);
+            $("#add_owner").submit();
+          }
+        });
+      }
+      else{
+        Materialize.toast("User "+ user +" could not be found", 4000)
+      }
+    });
+
+  });
+}
+
```

Comment:
This is the same block as `signs.js`, yes? We should be able to eliminate the redundancy.

This is what I'm thinking:

- Add a new js file: `add_owner_monkey_patch.js`.
- In that file, stub out a module, using our [preferred module pattern](https://kb-smc.chapman.edu/?p=1796):

```
/*
 * Long comment explaining things here.
 */
var AddOwnerMonkeyPatch = (function() {

    var onClick = function(event) {
        // Your code consolidated here. 
    };

    // Public API
    return {
        onClick: onClick
    };
});
```

Then, in these files, you can just update `fireWhenReady`, like so:

```

Utils.fireWhenReady(['slides#show'], function(e) {
    // TODO: Remove this line when materialize is fixed, etc...
    $('#add_owner').on('click', AddOwnerMonkeyPatch.onClick);

    $('.modal-trigger').modal();
});
```

Feel free to discuss this with me.